src/tests/ftest/object/create_many_dkeys.yaml

```
@@ -4,13 +4,13 @@ hosts:
   test_servers:
     - server-A
     - server-B
-timeout: 8000
+timeout: 3000
 server_config:
   name: daos_server
 pool:
   mode: 511
   name: daos_server
-  scm_size: 10G
+  scm_size: 8000000000
```

```suggestion
  scm_size: 8G
```
It's better to use the human readable.